_docs/05.1-contributors.html

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
@ananavati @raydecastro I used html here instead of markdown because you can't create 2 columns in markdown or make any of the elements interactive and styled. This isn't an html file that is attached to an original markdown file.

The 05.1-contributors.html file gets feed into the _layouts/contributing.html and then that goes into the main default.html.